frontend/package.json

```diff
@@ -67,84 +67,83 @@
+    "react": "^17.0.2",
+    "react-autosize-textarea": "^7.1.0",
     "react-chartjs-2": "^2.11.1",
-    "react-csv": "^2.0.1",
-    "react-datalist-input": "^1.2.12",
+    "react-csv": "^2.0.3",
     "react-datepicker": "^2.10.1",
-    "react-day-picker": "7.4.8",
-    "react-dom": "^17.0.1",
-    "react-focus-lock": "2.5.0",
-    "react-leaflet": "^3.1.0",
+    "react-day-picker": "7.4.10",
+    "react-dom": "^17.0.2",
+    "react-focus-lock": "2.5.1",
+    "react-leaflet": "^3.2.0",
     "react-leaflet-markercluster": "^3.0.0-rc1",
-    "react-meta-tags": "0.7.4",
-    "react-router-dom": "^5.1.2",
+    "react-meta-tags": "1.0.1",
+    "react-router-dom": "^5.2.0",
     "react-select": "^3.0.8",
     "react-spring": "^8.0.27",
-    "react-tooltip": "^4.2.13",
+    "react-tooltip": "^4.2.19",
     "seamless-scroll-polyfill": "1.2.3",
-    "styled-components": "5.2.1",
-    "tslib": "^2.1.0"
+    "styled-components": "5.3.0",
+    "tslib": "^2.2.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.13.10",
-    "@babel/preset-env": "^7.13.10",
-    "@sentry/webpack-plugin": "^1.14.2",
+    "@babel/core": "^7.14.3",
+    "@babel/preset-env": "^7.14.2",
+    "@sentry/webpack-plugin": "^1.15.1",
     "@storybook/addon-actions": "^6.1.21",
     "@storybook/react": "^6.1.21",
-    "@types/classnames": "^2.2.11",
-    "@types/jest": "^26.0.20",
-    "@types/leaflet": "^1.5.21",
-    "@types/lodash": "^4.14.168",
-    "@types/node": "^14.14.31",
-    "@types/react": "^17.0.1",
+    "@types/classnames": "^2.3.1",
+    "@types/jest": "^26.0.23",
+    "@types/leaflet": "^1.7.0",
+    "@types/lodash": "^4.14.169",
+    "@types/node": "^14.17.0",
+    "@types/react": "^17.0.6",
     "@types/react-csv": "^1.1.1",
     "@types/react-datepicker": "2.11.0",
-    "@types/react-dom": "^17.0.0",
+    "@types/react-dom": "^17.0.5",
     "@types/react-router-dom": "^5.1.7",
     "@types/react-select": "3.0.22",
-    "@types/react-tooltip": "^3.11.0",
-    "@types/styled-components": "^5.1.7",
-    "@typescript-eslint/eslint-plugin": "^4.22.0",
-    "@typescript-eslint/parser": "^4.22.0",
+    "@types/react-tooltip": "^4.2.4",
+    "@types/styled-components": "^5.1.9",
+    "@typescript-eslint/eslint-plugin": "^4.24.0",
+    "@typescript-eslint/parser": "^4.24.0",
     "babel-loader": "^8.2.2",
     "bulma": "0.7.5",
-    "css-loader": "^5.1.3",
-    "csstype": "^3.0.6",
-    "eslint": "^7.24.0",
-    "eslint-config-prettier": "^8.2.0",
+    "css-loader": "^5.2.4",
+    "csstype": "^3.0.8",
+    "eslint": "^7.26.0",
+    "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-prettier": "^3.4.0",
     "eslint-plugin-promise": "^5.1.0",
     "eslint-plugin-react": "^7.23.2",
     "eslint-plugin-react-hooks": "^4.2.0",
     "express": "^4.17.1",
     "express-http-proxy": "^1.6.2",
-    "fork-ts-checker-webpack-plugin": "^6.2.0",
+    "fork-ts-checker-webpack-plugin": "^6.2.10",
     "html-webpack-plugin": "^5.3.1",
-    "husky": "^5.0.9",
+    "husky": "^6.0.0",
```

Comment:
We are not using husky, are we? 🤔

Reply:
True, it's a leftover from ancient history :sweat_smile: I'll remove it